chart/templates/cdn-invalidator-sub/configmap.yaml

```diff
@@ -32,4 +32,8 @@ data:
   WORKSPACE_POSTGRES_HOST: {{ include "carto.postgresql.host" . }}
   WORKSPACE_POSTGRES_PORT: {{ include "carto.postgresql.port" . }}
   WORKSPACE_POSTGRES_USER: {{ include "carto.postgresql.user" . }}
+  WORKSPACE_POSTGRES_SSL_ENABLED: {{ .Values.externalPostgresql.sslEnabled | quote }}
+  {{- if and .Values.externalPostgresql.sslEnabled .Values.externalPostgresql.sslCA  }}
```

Comment:
I think that we also should allow to use that config with the internalPostgresql but implement it could be a timeConsuming task... :thinking: ...

Reply:
Maybe in the future, to complex to do it ATM